HomeTask1/Program.cs

```diff
@@ -1,3 +1,133 @@
-﻿// See https://aka.ms/new-console-template for more information
-string name = Console.ReadLine();
-Console.WriteLine("Hello," + name + "!");
+﻿using System;
+using System.Transactions;
+using System.Xml.Linq;
+
+namespace MyApp
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Lesson1_Task1();
+            Lesson2_Task1_Calculator();
+            Lesson2_Task2_RangeDetermination();
+            Lesson2_Task3_RangeDetermination();
+            Lesson2_Task4_ParityCheck();
+            Lesson2_Task4_1_ParityCheck();
+        }
+        public static void Lesson1_Task1()
+        {
+            string? name = Console.ReadLine();
```

Comment:
what is string? could you explain?

Reply:
It was "Warning" for Console.ReadLine(), so VisualStudio recommended to add ? after string. As I understand it is necessary for the compiler